lighthouse-core/runner.js

```diff
@@ -415,42 +406,44 @@ class Runner {
 
   /**
    * Get runtime configuration specified by the flags
-   * @param {!LH.ConfigSettings} settings
-   * @return {!Object} runtime config
+   * @param {LH.Config.Settings} settings
+   * @return {LH.Result.RuntimeConfig}
```

Comment:
it's possible we want to just move this to the report generator now that we have more readable throttling information in config settings?

Reply:
agree. not much left in this fn.

Reply:
yeah, we should just output the settings used into the LHR :D

Reply:
added to #4333